policyengine_us/parameters/gov/states/co/hcpf/chp/child.yaml

```
@@ -0,0 +1,20 @@
+description: Colorado requires a child to be in this age range to be eligible for the CHP+.
```

Similarly spell out CHP+ (all acronyms) in descriptions
```suggestion
description: Colorado limits eligibility for its Child Health Plan Plus to children in this age range.
```